lib/rules/__tests__/index.test.mjs

```diff
@@ -46,7 +46,8 @@ describe('fixable rules', () => {
 	});
 });
 
-describe('deprecated rules', () => {
+// eslint-disable-next-line jest/no-disabled-tests --- Leaving for future cases of deprecated rules.
+describe.skip('deprecated rules', () => {
```

Comment:
[ask] Is it possible to remove `skip` via mocking like `jest.mock('../rules/block-no-empty')`?

Reply:
Right now, the test checks the README for each rule; would the idea be to mock the rule's README? It feels a bit strange since it doesn't feel like the mock helps test the real behaviour at all (whether or not we documented the deprecation within the rule's README).

Reply:
Ah, that makes sense. 👍🏼 Let's leave it as-is.